src/DataCollector/MultiAuthCollector.php

```diff
@@ -114,8 +114,8 @@ protected function getUserInformation($user = null)
 
         // The default auth identifer is the ID number, which isn't all that
         // useful. Try username and email.
-        $identifier = $user instanceof Authenticatable ? $user->getAuthIdentifier() : $user->id;
-        if (is_numeric($identifier)) {
+        $identifier = $user instanceof Authenticatable ? $user->getAuthIdentifier() : $user->getKey();
+        if (is_numeric($identifier) || Str::isUuid($identifier) || Str::isUlid($identifier)) {
             try {
                 if (isset($user->username)) {
```

Comment:
`username` was change to `name`  since long time
An extra condition should be added.
```php
if (isset($user->name)) {
    $identifier = $user->name; // added
} elseif (isset($user->username)) {
    $identifier = $user->username;
} elseif (isset($user->email)) {
    $identifier = $user->email;
}
```

[https://github.com/laravel/laravel/database/migrations/2014_10_12_000000_create_users_table.php#L16](https://github.com/laravel/laravel/blob/6f6cfb020414bb084eaffeb910ac013095949e22/database/migrations/2014_10_12_000000_create_users_table.php#L16)

Reply:
Yes was changed, but the priority is email, name could be a long string, so, it was avoided on purpose, 
also two or more user could has the same name, username and email is always unique, the last commit must be reversed, the other looks good to me

Reply:
That makes a lot of sense

@abordage sorry, I was wrong on this part.